src/Microsoft.SqlTools.ServiceLayer/ServiceHost.cs

```diff
@@ -190,15 +190,15 @@ public void RegisterInitializeTask(InitializeCallback callback)
             );            
         }
 
-        private static ConnectionProviderOptions BuildConnectionProviderOptions()
+        internal static ConnectionProviderOptions BuildConnectionProviderOptions()
         {
             return new ConnectionProviderOptions
             {
                 Options = new ConnectionOption[]
                 {
                     new ConnectionOption
                     {
-                        Name = "Server Name",
+                        Name = "server",
```

Comment:
If we are fixing the Name field we should add an explicit DisplayName field for the UI to show.  By default the DMP handlers will mirror Name into DisplayName is there is no DisplayName provided.